src/routes.tsx

```diff
@@ -99,6 +100,7 @@ export const getAppRoutes = (): AppRouteConfig[] => {
     { routes: myCollectionInsightsRoutes },
     { routes: myCollectionInsightsCollectorProfileRoutes },
     { routes: newForYouRoutes },
+    { routes: newWorksFromGalleriesYouFollowRoutes },
```

Comment:
There's a danger warning around adding a smoke test that needs to be resolved before this is merged (unless its strictly logged-in-only and we can't)

Reply:
It's strictly logged-in only, but I've added a smoke test anyway to ensure that the page renders (only checking the title) when opening the URL.